st2common/st2common/services/action.py

```diff
@@ -64,7 +63,7 @@ def schedule(liveaction):
     # Validate action parameters.
     schema = util_schema.get_parameter_schema(action_db)
     validator = util_schema.get_validator()
-    jsonschema.validate(liveaction.parameters, schema, validator)
+    util_schema.validate(liveaction.parameters, schema, validator, use_default=True)
```

Comment:
For now, I only used it for validation action parameters, but eventually, we can also use it in all other places.